src/pages/waiting-room-to-car/cat-be/components/vehicle-checks-modal/vehicle-checks-modal.cat-be.page.html

```diff
@@ -18,11 +18,28 @@
     </ion-card-header>
     <ion-card-content>
       <vehicle-checks-question [formGroup]="formGroup"
-        [questionResult]="vehicleChecksQuestionResults"
+        [questionResult]="(pageState.showMeQuestions$ | async)[0]"
```

Comment:
Would be good to define a constant variable which is set to 3 in the case of show me question. And with a for loop display all these `vehicle-checks-question`. And use the same variable in order to create the initial state of the `vehicleChecksCatBEReducer`, so that the 2 never goes out of sync

Reply:
changed as suggested